app/controllers/admin/community_customizations_controller.rb

```diff
@@ -54,6 +56,25 @@ def update_details
 
   private
 
+  def validate_and_set_locales(params)
```

Comment:
The `and` in the name is a strong indicator that this method should be splitted :)